ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/util/BeaconStateUtil.java

```diff
@@ -419,6 +419,14 @@ public static UnsignedLong get_previous_epoch(BeaconState state) {
     else return get_current_epoch(state);
   }
 
+  /**
+   * Return the next ``epoch`` of the given state.
+   *
+   * <p>This method is no longer in the spec as of v0.4, but is retained here for convenience.
+   *
```

Comment:
I'm wondering if it was ever in the spec to begin with.  I think it *might* have been added by @dangerousfood  for convenience.  https://github.com/PegaSysEng/artemis/pull/231.  If this is true, then the comment that it is no longer needed in 0.4 possibly should be changed to something that explains that we deviated from the spec for convenience.  Just a suggestion.  Change if you want to.  I am approving regardless.

Reply:
I'm fairly certain it was there at one point, though it's not significant enough to track down now :-)